juanevillam/xtrovert
# Implement Auth Module with Locale Switcher

## Overview  
This PR implements the initial authentication module, including a locale switcher, authentication layout with header and footer, and brand logos.

---

## Key Changes  

### **Features**  
- Implemented authentication layout and page with header and footer.  
- Added brand logos (e.g., Facebook).  
- Integrated locale switcher for internationalization support.  

---

## Reasons for Changes  
These changes establish the foundational structure of the authentication module and introduce essential features like the locale switcher to support multiple languages, improving user accessibility.

---

## How to Test  
1. Run the application and navigate to the authentication page.  
2. Verify that the authentication layout displays correctly with the header and footer.  
3. Test the locale switcher to ensure it changes the language and updates the URL accordingly.  
4. Check responsiveness on different devices and screen sizes.  
5. Confirm that brand logos are displayed correctly.
